src/dev/jest/config.js

```diff
@@ -37,15 +37,23 @@ export default {
     '<rootDir>/src/test_utils',
     '<rootDir>/test/functional/services/remote',
   ],
+
   collectCoverageFrom: [
     'packages/kbn-ui-framework/src/components/**/*.js',
     '!packages/kbn-ui-framework/src/components/index.js',
     '!packages/kbn-ui-framework/src/components/**/*/index.js',
     'packages/kbn-ui-framework/src/services/**/*.js',
     '!packages/kbn-ui-framework/src/services/index.js',
     '!packages/kbn-ui-framework/src/services/**/*/index.js',
-    'src/legacy/core_plugins/metrics/**/*.js'
+    'src/legacy/core_plugins/metrics/**/*.js',
```

Comment:
should we just add a line for all core plugins ? 
`src/legacy/core_plugins/**/*.js`

Reply:
Yea, I can do that. The negative is that you can't really take meaning out of the summarized numbers any more because you are including code where coverage is still in mocha.  But I suppose as we continue to replace mocha with jest, this will be a more helpful number.